style.css

```diff
@@ -0,0 +1,66 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700;800&display=swap");
+
+@media only screen and (max-device-width: 375px) {
+  div#wrapper {
+    width: 400px;
+  }
+}
+
+div {
+  width: 16px;
+  height: 0;
+  background-color: #172b43;
+  margin: 4px 0;
+  border-radius: 30%;
+  border: 1px solid #172b43;
+}
+
+.menu_icon {
+  flex: auto;
+  padding-left: 85%;
+  padding-right: 30%;
+}
+
+.heading_up {
+  color: #172b4d;
+  text-align: center;
+  font-family: Inter, sans-serif;
+  font-size: 40px;
+}
```

Comment:
- [ ] As requested by the previous reviewer, Kindly use flex property to align your header items. ✔️